### ISSUE TITLE
BTW any advice to use dvb-ip-mpe2ts multiple time?

### ISSUE THREAD

**Issue by picatchoof** [piratfm/eti-tools#49]
Hello again.
Sorry if I bather you.
I'm not well skilled in development and Linux scripting.
I have one OUTPUT ts file either as fifo ts or even as nc output file.
This TS content has 10 or maybe more  streams under DVB-MPE encapsulation.

I can easily extract one stream.

I need to apply the dvb-ip-mpe2ts 10 times on the source file in order to extract the 10 streams in it. And save them either as fifo or stream them over the LAN.

Any advice about the best approach (Piping, tee, source code customization....)

Best regards

**Comment by andimik** [comment]
What about a shell script?

**Comment by picatchoof** [comment]
Yes it may be a solution but what to put as list of CLI that includes dvb-ip-mpe2ts and his options.
The issue is that all the ten command must be run in order to do live streaming from the source to the other side.

Can you please give an idea about what can be the SH script file?

BR

**Comment by andimik** [comment]
Have you tried to use all 10 commands line per line in the sh file?

**Comment by picatchoof** [comment]
Yes it can be a solution to put 10 lines dvb-ip-mpe2ts command lines.
But the issue is that.
1. If the input of the first is the FIFO that comes from other application the other dvb-ip-mpe2ts won't be able to get information from the input since it will be emptied by the first line also the file will be locked by the first dvb-ip-mpe2ts.

2. If the input comes from nc so the listing port will be locked by the first dvb-ip-mpe2ts and the others won't be able to use it.
